File: exam/src/main/java/in/testpress/exam/ui/SearchActivity.java
package in.testpress.exam.ui;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.WindowManager;

import in.testpress.exam.R;

public class SearchActivity extends AppCompatActivity {

    @Override
    public void onCreate(final Bundle savedInstanceState) {
        getWindow().setFlags(WindowManager.LayoutParams.FLAG_SECURE, WindowManager.LayoutParams.FLAG_SECURE);
        super.onCreate(savedInstanceState);
        setContentView(R.layout.testpress_container_layout_without_toolbar);
        SearchFragment searchFragment = new SearchFragment();
        searchFragment.setArguments(getIntent().getExtras());
        getSupportFragmentManager().beginTransaction()
                .replace(R.id.fragment_container, searchFragment).commitAllowingStateLoss();
    }

}

File: exam/src/main/java/in/testpress/exam/ui/SearchActivity.kt
package `in`.testpress.exam.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import `in`.testpress.exam.R

class SearchActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.testpress_container_layout_without_toolbar)
        val searchFragment = SearchFragment().apply {
            arguments = intent.extras
        }
        supportFragmentManager.beginTransaction()
            .replace(R.id.fragment_container, searchFragment).commitAllowingStateLoss()
    }
}